build.gradle

```diff
@@ -111,6 +114,9 @@ dependencies {
 	testCompile 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
 	testCompile 'org.junit.jupiter:junit-jupiter-api'
 
+	// temporary - gradle bug prevents getting this transitively
```

Comment:
Can we try to avoid this with the latest Gradle `6.2.2`?
Just in case it has been fixed over there somehow...

Reply:
No; not fixed yet - discussed with the gradle team this morning,